uobtheatre/reports/views.py

```diff
@@ -0,0 +1,177 @@
+from django.http import HttpResponse
+from xlsxwriter.utility import xl_rowcol_to_cell
+
+from uobtheatre.reports.utils import ExcelReport
+
+from . import reports
+
+
+def period_totals(request, start_time, end_time):
```

Comment:
I think these views are doing too much. We should move this logic somewhere else